packages/bierzo-wallet/src/config/index.ts

```diff
@@ -64,7 +64,7 @@ interface WindowWithConfig extends Window {
 
 const loadConfigurationFile = async (): Promise<Config> => {
   if (process.env.NODE_ENV === "test") {
-    return (window as WindowWithConfig).developmentConfig;
+    return ((window as Window) as WindowWithConfig).developmentConfig;
```

Comment:
What if we will wrap this into something like `getWindowWithConfig(window)` helper where we will implement all of this type castings?

Reply:
We can also avoid the argument and use the global object `window` from inside `function getWindowWithConfig(): WindowWithConfig`